models/issues/content_history.go

```diff
@@ -167,7 +167,20 @@ func FetchIssueContentHistoryList(dbCtx context.Context, issueID int64, commentI
 	return res, nil
 }
 
-//SoftDeleteIssueContentHistory soft delete
+// HasIssueContentHistory check if a ContentHistory entry exists
+func HasIssueContentHistory(dbCtx context.Context, issueID int64, commentID int64) (bool, error) {
+	exists, err := db.GetEngine(dbCtx).Cols("id").Exist(&ContentHistory{
+		IssueID:   issueID,
+		CommentID: commentID,
+	})
+	if err != nil {
+		log.Error("can not fetch issue content history. err=%v", err)
```

Comment:
Why not just return the error but add a special log here?

Reply:
I think it just follows my first PR, a personal preference for error handling.

Usually I would like to report important errors as early as possible. 

Otherwise:
1. The error may be ignored be callers
2. The caller prints the error with a unrelated stacktrace (especially in Go, the errors don't have stacktrace)